lib/__tests__/disableRanges.test.mjs

```diff
@@ -953,8 +1062,7 @@ it('Less // line-disabling comment (with description)', async () => {
 	});
 });
 
-// eslint-disable-next-line jest/no-disabled-tests -- see #5348
```

Comment:
This issue was fixed by #5458 in version 14.0.0-0.
i.e. one day before the release of 14.0.0
We can close #5348.

Reply:
Thank you for digging this up @Mouvedia 🙇